src/test/java/com/echobox/api/linkedin/jsonmapper/LinkedInJsonMapperTest.java

```diff
@@ -0,0 +1,191 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.echobox.api.linkedin.jsonmapper;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import com.echobox.api.linkedin.jsonmapper.JsonMapper.JsonMappingCompleted;
+import com.echobox.api.linkedin.types.CodeAndNameType;
+import com.echobox.api.linkedin.types.Company;
+import com.echobox.api.linkedin.types.Location;
+
+import lombok.Getter;
+
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * DefaultJsonMapperTest
+ * @author Joanna
+ *
+ */
+public class LinkedInJsonMapperTest {
```

Comment:
Rename the test as well?
```suggestion
public class LinkedInJsonMapperV1Test {
```